### ISSUE TITLE
`/admin/teams` page comes from Teams table, not Students table

### ISSUE THREAD

**Issue by pconrad** [ucsb-cs156-s21/proj-mapache-search#165]
After #162 and #164 change the /admin/teams page so that it gets its information from the teams page instead of a query based on the students table.

**Comment by andrewhlu** [comment]
Closing as I think this issue is already done.
(If that's incorrect, let me know and/or file a new issue with a more clear description)
